src/Sylius/Behat/Resources/config/suites/api/taxon/managing_taxons.yml

```diff
@@ -7,17 +7,18 @@ default:
             contexts:
                 - sylius.behat.context.hook.doctrine_orm
 
+                - sylius.behat.context.transform.channel
                 - sylius.behat.context.transform.locale
                 - sylius.behat.context.transform.shared_storage
                 - sylius.behat.context.transform.taxon
 
                 - sylius.behat.context.setup.channel
                 - sylius.behat.context.setup.locale
                 - sylius.behat.context.setup.admin_security
+                - sylius.behat.context.setup.admin_api_security
                 - sylius.behat.context.setup.taxonomy
 
                 - sylius.behat.context.api.admin.managing_taxons
-
             filters:
-                tags: "@managing_taxons&&@api"
+                tags: "@managing_taxons&&@api@ajax"
```

Comment:
```suggestion
                tags: "@managing_taxons&&@api"
```

Reply:
@dawkaa I've checked that and because of that line, these behat scenarios are not executed for the API :/ You can test it out on your own by removing `@ui @javascript` temporarily, leaving `@api`. Then run `vendor/bin/behat features/taxonomy/managing_taxons/deleting_taxon.feature`.

In my case it results with:
![image](https://user-images.githubusercontent.com/13316080/220576898-bdd88e82-a9c0-4942-821b-b82bfba74935.png)